meta-ros-common/conf/ros-distro/ros-distro.conf

```diff
@@ -10,11 +10,11 @@ ROS_BUILD_TYPE ??= "unset"
 
 # ROS_DISTRO_COMPAT contains the list of ROS_DISTRO-s with which this branch of meta-ros is compatible (based on the versions of
 # the platform packages provided by it and the branches of the upstream OE layers used).
-ROS_DISTRO_COMPAT = "melodic dashing eloquent"
+ROS_DISTRO_COMPAT = "melodic dashing eloquent iron"
 
 # This variable is updated with the new release series as the first commit to [master] after the branch for the current release
 # series is created.
-ROS_OE_RELEASE_SERIES = "langdale"
+ROS_OE_RELEASE_SERIES = "kirkstone"
```

Comment:
@codewithpassion I don't think we want this change if meant to be merged into the `langdale` branch.